config/locales/en.yml

```
@@ -2886,7 +2887,7 @@ en:
             one: To see if you qualify, we need more information about you.
             other: To see if you qualify, we need more information about you and your household.
           select_household_members: 'Select the members of the household any of these situations applied to in %{tax_year}:'
-          situation_incarceration: experienced incarceration
+          situation_incarceration: Were incarcerated
```

these are concatenated as a part of other sentences so should not be capitalized right?
```suggestion
          situation_incarceration: were incarcerated
```